File: Pascal/fibonacci.pas
VAR  m,p:integer;

function fib(n:integer):integer;
begin
    if n=0 then fib:=0
        else if n=1 then fib:=1
            else fib:=fib(n-1)+fib(n-2);
end;
BEGIN
    readln(m);
    p:=fib(m);
    writeln('fib(',m,')=',p)
END.

File: Go/fibonacci.go
package main

import "fmt"

func fib(index int) int {
	if index == 0 {
		return 0
	}
	if index == 1 {
		return 1
	}
	return fib(index-1) + fib(index-2)
}

func main() {
	fmt.Println(fib(20))
}
